src/game/fleet/fleet.rs

```diff
@@ -0,0 +1,61 @@
+use actix_web::{get, post, web, HttpResponse};
+use serde::{Serialize, Deserialize};
+use uuid::Uuid;
+use crate::{
+    lib::{
+        Result,
+        error::{InternalError},
+        auth::Claims
+    },
+    game::{
+        game::{GameID, GameDataMessage},
+        player::PlayerID,
+        system::SystemID
+    },
+    AppState
+};
+
+const FLEET_COST: usize = 10;
+
+#[derive(Serialize, Deserialize, Clone, Hash, PartialEq, Eq, Copy)]
+pub struct FleetID(Uuid);
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct Fleet{
+    id: FleetID,
+    system: SystemID,
+    player: PlayerID,
+}
+#[derive(Deserialize)]
+pub struct FleetCreationData {
+    system: SystemID
+}
+
+#[post("/")]
+pub async fn create_fleet(state: web::Data<AppState>, json_data: web::Json<FleetCreationData>, info: web::Path<(GameID,)>, claims: Claims) -> Result<HttpResponse> {
```

Comment:
Maybe we should add a maximum line width ?
I usualy use 80 columns max, but it is as you wish.

Reply:
Also comments... I'd like someone to add comments especially because when we will use a database, the code will need to be understandable in order to keep the same behavior.